utils/envvars/parser.go

```diff
@@ -141,6 +148,35 @@ func parseStringDuration(value string) (time.Duration, error) {
 	return d, nil
 }
 
+// GetSAASHost returns SAAS host env variable value if it's valid.
+// Otherwise returns defaultSAASHost.
+func GetSAASHost(env string) (string, error) {
+	if v, ok := os.LookupEnv(envSAASHost); ok {
+		if ok := govalidator.IsURL(v); ok {
+			logrus.Warnf("SAAS host changed to %q.", v)
+
+			return v, nil
+		}
+
+		return v, fmt.Errorf("environment variable %q has invalid format %q. Expected {host}:{port}", env, v)
+	}
+
+	if v, ok := os.LookupEnv(env); ok {
+		if ok := govalidator.IsURL(v); ok {
```

Comment:
But that's not an URL, it is just `host:port`. Use `net.SplitHostPort` and check that port is present, and there is not error.

Reply:
done, thanks